src/index.mjs

```diff
@@ -1,8 +1,8 @@
 export * as constants from './constants.mjs';
 export { default as generators } from './generators/index.mjs';
 export { default as createGenerator } from './generators.mjs';
-export { default as createLoader } from './loader.mjs';
+export * from './loader.mjs';
```

Comment:
Instead of this, can you make a new folder called loaders and have a file for the js one and one for the markdown one. Same for parsers.

Reply:
This allows us to separate the code for markdown loader/parser from the js ones :)